frontend/src/components/form/TextArea.tsx

```diff
@@ -1,41 +1,62 @@
-import styled from "@emotion/styled";
 import { forwardRef } from "react";
 import { MessageDescriptor, useIntl } from "react-intl";
-
-const StyledTextarea = styled.textarea`
-  width: 100%;
-  min-height: 10rem;
-
-  padding: 0.25rem 0.5rem;
-`;
+import { Field, Textarea as ChakraTextarea } from "@chakra-ui/react";
+import styled from "@emotion/styled";
 
 const InputWrapper = styled.label`
   display: block;
   margin-bottom: 1rem;
 `;
 
-const Label = styled.span`
-  display: block;
-  margin-bottom: 0.25rem;
+const styledTextareaStyles = {
+  padding: "0.25rem 0.5rem",
+  minHeight: "10rem",
+  width: "100%",
+};
+
+const ErrorMessage = styled.div`
+  color: var(--error-color);
+  font-size: 0.875rem;
+  margin-top: 0.25rem;
 `;
 
 interface Props {
   label: MessageDescriptor;
-  children: React.ReactNode;
+  helperText?: React.ReactNode;
+  errorText?: React.ReactNode;
+  invalid?: boolean;
+  size?: "xs" | "sm" | "md" | "lg" | "xl";
```

Comment:
I've seen this now defined in a few places. Can we just point to the corresponding Chakra UI type definition instead of repeating it in multiple places? 🤔